test/fluree/db/query/misc_queries_test.clj

```diff
@@ -97,73 +95,78 @@
+                               :schema/email "alice@flur.ee"
+                               :schema/age   42}
+                              {:id          :ex/bob,
+                               :type        :ex/User,
+                               :schema/name "Bob"
+                               :schema/age  22}
+                              {:id           :ex/jane,
+                               :type         :ex/User,
+                               :schema/name  "Jane"
+                               :schema/email "jane@flur.ee"
+                               :schema/age   30}]}})]
       (testing "Query that pulls entire database."
-        (is (= [[:ex/jane :id "http://example.org/ns/jane"]
-                [:ex/jane :type :ex/User]
-                [:ex/jane :schema/name "Jane"]
-                [:ex/jane :schema/email "jane@flur.ee"]
-                [:ex/jane :schema/age 30]
-                [:ex/bob :id "http://example.org/ns/bob"]
-                [:ex/bob :type :ex/User]
-                [:ex/bob :schema/name "Bob"]
-                [:ex/bob :schema/age 22]
-                [:ex/alice :id "http://example.org/ns/alice"]
-                [:ex/alice :type :ex/User]
-                [:ex/alice :schema/name "Alice"]
-                [:ex/alice :schema/email "alice@flur.ee"]
-                [:ex/alice :schema/age 42]
-                [:schema/age :id "http://schema.org/age"]
-                [:schema/email :id "http://schema.org/email"]
-                [:schema/name :id "http://schema.org/name"]
-                [:ex/User :id "http://example.org/ns/User"]
-                [:rdfs/Class :id "http://www.w3.org/2000/01/rdf-schema#Class"]
-                [:type :id "@type"]
-                [:id :id "@id"]]
-               @(fluree/query db {:select ['?s '?p '?o]
-                                  :where  {:id '?s
-                                           '?p '?o}}))
+        (is (= #{[:ex/jane :id "http://example.org/ns/jane"]
+                 [:ex/jane :type :ex/User]
+                 [:ex/jane :schema/name "Jane"]
+                 [:ex/jane :schema/email "jane@flur.ee"]
+                 [:ex/jane :schema/age 30]
+                 [:ex/bob :id "http://example.org/ns/bob"]
+                 [:ex/bob :type :ex/User]
+                 [:ex/bob :schema/name "Bob"]
+                 [:ex/bob :schema/age 22]
+                 [:ex/alice :id "http://example.org/ns/alice"]
+                 [:ex/alice :type :ex/User]
+                 [:ex/alice :schema/name "Alice"]
+                 [:ex/alice :schema/email "alice@flur.ee"]
+                 [:ex/alice :schema/age 42]
+                 [:schema/age :id "http://schema.org/age"]
+                 [:schema/email :id "http://schema.org/email"]
+                 [:schema/name :id "http://schema.org/name"]
+                 [:ex/User :id "http://example.org/ns/User"]
+                 #_[:rdfs/Class :id "http://www.w3.org/2000/01/rdf-schema#Class"]
```

Comment:
What's up here?

Reply:
Oh, I forgot to delete that. We no longer include that flake in the base flakes, it will get created when a user references it in a transaction.